mm2src/coins/lp_coins.rs

```diff
@@ -312,6 +313,8 @@ pub type TxHistoryResult<T> = Result<T, MmError<TxHistoryError>>;
 pub type RawTransactionResult = Result<RawTransactionRes, MmError<RawTransactionError>>;
 pub type RawTransactionFut<'a> =
     Box<dyn Future<Item = RawTransactionRes, Error = MmError<RawTransactionError>> + Send + 'a>;
+pub type SignRawTransactionResult = Result<SignRawTransactionResponse, MmError<RawTransactionError>>;
+pub type SignEthTransactionResult = Result<SignEthTransactionResponse, MmError<RawTransactionError>>;
```

Comment:
These can be only one type since `SignRawTransactionResponse` is identical to `SignEthTransactionResponse`.